_posts/2023-01-24-jaydondulgar-introduction-meme.md

```diff
@@ -0,0 +1,7 @@
+---
+title:  Same Picture
+author: Jaydon
+image: pam.jpeg
+chapter: 2
+---
+The meme is rather self-explanatory. The terms "virtual machine" and "operating system" are interchangable
```

Comment:
I know the book says these are sometimes referring to the same thing, but if you talk about an OS as a virtual machine outside of using the OS's API, people are not going to know what you're talking about. Out of context, an operating system and a virtual machine are two very different things.